### DISCUSSION TITLE
Let's hack a cool proof of concept together!

### DISCUSSION THREAD

**Discussion by alexbit-codemod** [cytoscape/cytoscape.js#2864] [General]
Anyone here is interested in hacking a proof of concept together (a web app that is heavy on Cytoscape)?
If you are a builder, hustler, energetic, and find [my Linkedin profile](https://www.linkedin.com/in/alexbit/) legit, let's [chat](https://calendly.com/alex-from-stackgraph/15min)!

**Comment by GaltMidas** [comment]
Still up for this?  I'll ping your calendly if so.

**Reply by alexbit-codemod** [reply]
Hey Mike. Thanks for getting back to me. Now I'm good. Found some friends. But I'll keep you in mind for future ones. Cheers
